03-transactions.md

```diff
@@ -86,14 +86,14 @@ This output sends funds back to the owner of this commitment transaction, thus m
         # Penalty transaction
         <revocation-pubkey>
```

Comment:
Just to be sure, are you keeping dashes in `<revocation-pubkey>`, `<local-delayedkey>`, `<ripemd-of-payment_hash>` on purpose? 

Should we also fix inconsistencies between pubkey and key? We have both `<revocation-key>` and `<revocation-pubkey>`.

Ther are also inconsistencies between `<revocation-sig>` and `<revocationkey-hash>`. If that's outside the scope of this PR I'll submit another one.